changemakers/fixtures/state.json

```diff
@@ -278,5 +278,13 @@
   "doctype": "State",
   "modified": "2022-12-21 22:58:03.036242",
   "name": "Dadra and Nagar Haveli and Daman and Diu"
+ },
+ {
+  "code": null,
+  "country": "India",
+  "docstatus": 0,
+  "doctype": "State",
+  "modified": "2023-10-12 14:53:15.118309",
+  "name": "Nairobi"
```

Comment:
Let's remove this? 😅

Reply:
😂I dont even remember when I d that, but sure it is unnecessary. Let me wipe it